<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogcatDialog</class>
 <widget class="QDialog" name="LogcatDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logcat</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="layoutLogcat">
    <item row="1" column="1">
     <widget class="QCheckBox" name="checkBoxDebug">
      <property name="text">
       <string>Debug</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="checkBoxWarnings">
      <property name="text">
       <string>Warning</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="checkBoxInformations">
      <property name="text">
       <string>Information</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QCheckBox" name="checkBoxErrors">
      <property name="text">
       <string>Error</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="10">
     <widget class="QTableView" name="tableView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="5" column="1" colspan="10">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QCheckBox" name="checkBoxVerbose">
      <property name="text">
       <string>Verbose</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>filter</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Sender</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Message</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="10">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>5</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="11">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>5</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="7">
     <widget class="QSpinBox" name="spinBoxBufferLimit">
      <property name="maximum">
       <number>9999</number>
      </property>
     </widget>
    </item>
    <item row="1" column="7">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Buffer limit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QCheckBox" name="checkBoxAutoScroll">
      <property name="text">
       <string>autoscroll</string>
      </property>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="QPushButton" name="pushButtonClearLogcat">
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9">
     <widget class="QPushButton" name="saveButton">
      <property name="toolTip">
       <string>Save</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../zasoby.qrc">
        <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
